ovn-tester/cms/ovn_kubernetes/__init__.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2023 Canonical
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from collections import namedtuple
```

Comment:
I'm no Python expert so there are probably good reasons against this but what do you think of moving all this code to a new `ovn_kubernetes.py` file?  E.g., https://github.com/dceara/ovn-heater-1/commit/d9736879097b606d3807201a72e0ad3717cd2226